src/lib/babel-config.js

```diff
@@ -4,12 +4,8 @@ export default (env, options={}) => ({
 		[require.resolve('babel-preset-env'), {
 			loose: true,
 			modules: options.modules || false,
+			browsers: options.browsers,
```

Comment:
Via .babelrc or package.json (or even by changing webpack babel-loader query). I don't think you have to worry about that at all because babel is so smart that it extends over this configuration, i.e. a `.babelrc`that looks like this ```{ plugins: ["my-awesome-plugin"] }``` will get everything applied from this file + 'my-awesome-plugin'.

In case of collisions stuff from .babelrc takes precedence over our internal settings (don't remember if options were merged tough, e.g. when `env` preset was declared stuff like `loose: true` was preserved or in this case browsers would be preserved)